Topic 3/src/test/java/bootcamp/exercises/WordWrappingTest.java

```diff
@@ -0,0 +1,30 @@
+package bootcamp.exercises;
+
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+
+public class WordWrappingTest {
+
+	WordWrapping wordWrapping= new WordWrapping();
+	
+	@Test
+	public void test1() {		
+		assertEquals(wordWrapping.wordWrap("Hello World!",7),"Hello W/orld!");
```

Comment:
[High] This is not a valid scenario. The exercise says:
If the row-length is 7 and the input string is "Hello Word!", the result should be: "Hello" and "Word!"